src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vm/disk/_attach_detach_data_disk.py

```diff
@@ -11,6 +11,10 @@
 from azure.cli.core.aaz import *
 
 
+@register_command(
+    "vm disk attach-detach-data-disk",
+    is_preview=True,
+)
```

Comment:
The aaz repo is just the CodeGen model of CLI, and the logic of this CLI repo is actually what the users can use and see. You need to re-generate codes from the CodeGen UI, and the example will be display in this file.